src/BaGet/Program.cs

```diff
@@ -71,6 +71,8 @@ await provider
                     {
                         config.SetBasePath(root);
                     }
+
+                    config.AddKeyPerFile("/run/secrets", optional: true);
```

Comment:
What do you think of doing similar to the `BAGET_CONFIG_ROOT` environment variable - something like `BAGET_KEY_PER_FILE_PATH`. If the environment variable is set, we call `config.AddKeyPerFile(...);`. Does that seem reasonable?

Reply:
I can see the utility of allowing the directory to be configurable, but default to `/run/secrets`.

The `optional: true` parameter means the path is ignored if not present, so there should be no harm in calling this in all environments (no options values are provided in the event of path nonexistence or no files being present in the directory, and no exceptions are thrown by the provider). [Docker Secrets](https://docs.docker.com/engine/swarm/#how-docker-manages-secrets) and similar environments (including the now-legacy Rancher 1.6) use `/run/secrets` for secret value mountpoints by convention. Kubernetes environments are more flexible with the ability to select a mountpoint in a pod for a secret value.

One potential to note is that Kubernetes does support updating secret values in-situ on a deployed pod ([docs](https://kubernetes.io/docs/concepts/configuration/secret/#mounted-secrets-are-updated-automatically), so using `reload: true` as well might be useful - but that then requires the project to be aware of configuration fluidity, and use either `IOptionsMonitor<T>` for singleton services or `IOptionsSnapshot<T>` for scoped/transient services to receive the updated values.